arviz/plots/backends/matplotlib/elpdplot.py

```diff
@@ -179,7 +179,8 @@ def plot_elpd(
                     "{} - {}".format(models[i], models[j + 1]), fontsize=titlesize, wrap=True
                 )
         if xlabels:
-            set_xticklabels(ax[-1, -1], coord_labels)
+            for i in range(len(ax)):
+                set_xticklabels(ax[-1, :][i], coord_labels)
```

Comment:
I am not completely sure about what is causing the test error. Matplotlib locators do take steps as an argument, and there has been no breaking change on that. The only thing that would make sense would be for the loop to be over the last column instead of being over the bottom row so that `set_xticklabels` is trying to format an axis that has been set to `off` and thus has no locator. But then you would not have been able to generate the example image you shared, so I am completely at a loss here, no idea what is happening.

Reply:
Maybe we don't need to loop over the last row, the x axis is shared and should have the same labels for all plots automatically I think :thinking: